src/resources/appointment.ts

```diff
@@ -58,20 +58,17 @@ export interface AppointmentParameters {
     };
     type: string;
   };
-  meta?: {
-    booker?: number;
-    notify?: {
-      client?: boolean;
-      user?: boolean;
-    };
-    utm?: {
-      campaign?: string;
-      content?: string;
-      source?: string;
-      medium?: string;
-      term?: string;
-    };
+  meta?: AppointmentMeta;
+}
+
+export interface AppointmentMeta {
+  booker?: number;
+  notify?: {
+    client?: boolean;
+    user?: boolean;
   };
+  utm?: UtmParameters;
+  [key: string]: unknown;
```

Comment:
I think instead of unknown, we might want to instead use a union type, such as `string | number | boolean`